test/Makefile

```diff
@@ -138,7 +138,6 @@ test_srcs := \
 	sq-poll-share.c \
 	sqpoll-sleep.c \
 	sq-space_left.c \
-	statx.c \
```

Comment:
This doesn't look right?

Reply:
statx.c gets added if CONFIG_HAVE_STATX on line 161

Reply:
If the system doesn't have statx support, liburing should make its own definition. Maybe that got broken, in which case, that's what needs fixing. The statx test should always compile just fine.

Reply:
Ah ok, I'm mistaken, we do have a check for it and add it conditionally. I guess it all looks fine then, thanks.